automation/src/data/MockData.ts

```diff
@@ -0,0 +1,30 @@
+import faker from 'faker';
+
+/**
+ * Get fake user information.
+ */
+const user = {
```

Comment:
nit: I like to name my mocks like `mockUser` so that when tests start to get complex, you can tell what's real and what's not.

Reply:
renamed to `fakeUser`